core/controllers/suggestion.py

```diff
@@ -91,6 +92,51 @@ def put(self, target_id, suggestion_id):
         self.render_json(self.values)
 
 
+class SuggestionToTopicActionHandler(base.BaseHandler):
+    """Handles actions performed on suggestions to topics."""
+
+    ACTION_TYPE_ACCEPT = 'accept'
+    ACTION_TYPE_REJECT = 'reject'
+
+    @acl_decorators.get_decorator_for_accepting_suggestion(
+        acl_decorators.can_edit_topic)
+    def put(self, target_id, suggestion_id):
+        if not constants.USE_NEW_SUGGESTION_FRAMEWORK:
+            raise self.PageNotFoundException
+
+        if not feconf.ENABLE_NEW_STRUCTURES:
+            raise self.PageNotFoundException
+
+        if len(suggestion_id.split('.')) != 3:
+            raise self.InvalidInputException('Invalid format for suggestion_id.'
```

Comment:
Generally, break after '(' if you're doing a multi-line thing. This gives you more space to write, rather than you needing to bunch everything up to the right.

Ditto below.

Reply:
Done